### PR TITLE
Add Identicon component

### PR DESCRIPTION
**Pull request by sisou**

### CODE REVIEW

File: demos/Identicon.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Identicon</title>
+</head>
+<body>
+    <div id="identicons" style="flex-direction: row;"></div>
+
+<link href="../src/nimiq-style.css" rel="stylesheet">
+<link href="../src/components/Identicon.css" rel="stylesheet">
```
Comment (Bettelstab) [inline demos/Identicon.html]: That file does not exist

File: src/components/Identicon.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+class Identicon {
+
+    /**
+     * @param {string} [address]
+     */
+    constructor(address) {
+        this._address = address;
+        /** @type {HTMLImageElement} */
+        this._el;
```
Comment (danimoh) [inline src/components/Identicon.js]: We're using $el syntax for properties that are HTMLElements

File: src/components/Identicon.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    getElement() {
+        if (this._el) return this._el;
+
+        this._el = document.createElement('img');
```
Comment (danimoh) [inline src/components/Identicon.js]: DOM creation should be triggered already in constructor, see PinInput.js

File: src/lib/Iqons.js [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+    /**
+     * @param {string} text
+     */
+    static async toDataUrl(text) {
+        const base64string = btoa(await this.svg(text));
+        return `data:image/svg+xml;base64,${base64string.replace(/#/g, '%23')}`;
+    }
+
+    /**
```
Comment (danimoh) [inline src/lib/Iqons.js]: This can be removed in keyguard

File: src/lib/Iqons.js [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+        return `<svg viewBox="0 0 160 160" width="160" height="160" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/2000/xlink" >
+    <path fill="none" stroke="${color}" stroke-width="${2 * strokeWidth}" transform="translate(0, 8) scale(0.5)" d="M251.6 17.34l63.53 110.03c5.72 9.9 5.72 22.1 0 32L251.6 269.4c-5.7 9.9-16.27 16-27.7 16H96.83c-11.43 0-22-6.1-27.7-16L5.6 159.37c-5.7-9.9-5.7-22.1 0-32L69.14 17.34c5.72-9.9 16.28-16 27.7-16H223.9c11.43 0 22 6.1 27.7 16z"/>
+    <g transform="scale(0.9) translate(9, 8)">
+        <circle cx="80" cy="80" r="40" fill="none" stroke="${color}" stroke-width="${strokeWidth}" opacity=".9"></circle>
+        <g opacity=".1" fill="#010101"><path d="M119.21,80a39.46,39.46,0,0,1-67.13,28.13c10.36,2.33,36,3,49.82-14.28,10.39-12.47,8.31-33.23,4.16-43.26A39.35,39.35,0,0,1,119.21,80Z"/></g>\`
+    </g>
+</svg>`;
+    }
+
+    /**
```
Comment (danimoh) [inline src/lib/Iqons.js]: can be removed in keyguard

File: src/lib/Iqons.js (hunk 4 above)
Comment (sisou) [reply src/lib/Iqons.js]: Placeholders are set as the image now as long as no address is set.

File: src/lib/Iqons.js (hunk 5 above)
Comment (sisou) [reply src/lib/Iqons.js]: Placeholders are set as the image now as long as no address is set.

**Comment by sisou** [comment]
@danimoh @Bettelstab please review again

**Pull request merged**